Dockerfile

```diff
@@ -9,4 +11,29 @@ RUN apt-get update \
         snmp \
     && docker-php-ext-install -j$(nproc) mysqli snmp \
     && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
-    && docker-php-ext-install -j$(nproc) gd
\ No newline at end of file
+    && docker-php-ext-install -j$(nproc) gd \
+    && rm -rf /var/lib/apt/lists/* \
+    && apt-get clean
+
+RUN docker-php-ext-install zip
```

Comment:
A comment that the Composer setup starts here would be nice.
Because the `zip` extension is required by Composer, but not by LanSuite.